[14.0][DOC]Adjust the page migration_details:

jcdrubay opened the pull request:
- With titles to highlight the steps and extra info added within the step
- Add details for installations
- Add some links to module coverage status and development page

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:




--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

StefanRijnhart commented on doc/source/migration_details.rst:
```diff
@@ -5,47 +5,64 @@ Check out the code manually and upgrade your database by calling odoo-bin,
-4. Edit the configuration files and command line parameters to point to the
-   database you are going to upgrade. Run Odoo with the *--update all
-   --stop-after-init --load=web,openupgrade_framework* flags.
+OpenUpgrade repository includes both `openupgrade_framework`and `openupgrade_scripts`:
```
Backtick confusion

![image](https://user-images.githubusercontent.com/1033124/111758177-768a8f80-889c-11eb-9060-a12011a28113.png)


```suggestion
The OpenUpgrade repository includes both `openupgrade_framework` and `openupgrade_scripts`:
```

StefanRijnhart commented on doc/source/migration_details.rst:
```diff
@@ -5,47 +5,64 @@ Check out the code manually and upgrade your database by calling odoo-bin,
+Check if there are migration scripts provided for the set of modules that
+are installed in your Odoo database. If there are modules for which no
+migration scripts have been developed yet, your migration may fail or the
+integrity of your database may me lacking. Check the module coverage in
```
```suggestion
integrity of your database may be lacking. Check the module coverage in
```

StefanRijnhart commented on doc/source/migration_details.rst:
```diff
@@ -5,47 +5,64 @@ Check out the code manually and upgrade your database by calling odoo-bin,
+are installed in your Odoo database. If there are modules for which no
+migration scripts have been developed yet, your migration may fail or the
+integrity of your database may me lacking. Check the module coverage in
+this documentation in :doc:`status` and refer to the :doc:`development`
```
```suggestion
this documentation under :doc:`status` and refer to the :doc:`development`
```

StefanRijnhart commented on doc/source/migration_details.rst:
```diff
@@ -5,47 +5,64 @@ Check out the code manually and upgrade your database by calling odoo-bin,
+migration scripts have been developed yet, your migration may fail or the
+integrity of your database may me lacking. Check the module coverage in
+this documentation in :doc:`status` and refer to the :doc:`development`
+documentation to add the missing migration modules.
```
```suggestion
documentation to add the missing migration scripts.
```

StefanRijnhart commented on doc/source/migration_details.rst:
```diff
@@ -5,47 +5,64 @@ Check out the code manually and upgrade your database by calling odoo-bin,
+backup of your live database before you start this process!
+
+It is easy to do through the database manager of your old odoo
+`{your.old.odoo}/web/database/manager`
```
Maybe it is me but I associate using the database manager for backups with a lower knowledge level than needed to run OpenUpgrade properly. I would personally leave this out because if people need us to tell them how to make a proper backup of their setup, their voyage in OpenUpgrade will not end well.

jcdrubay replied:
I imagine the same :)

StefanRijnhart commented on doc/source/migration_details.rst:
```diff
@@ -5,47 +5,64 @@ Check out the code manually and upgrade your database by calling odoo-bin,
+****************************************************
+
+Edit the configuration files and command line parameters to point to the
+database you are going to upgrade. Run Odoo with the ``--update all
```
```suggestion
database you are going to upgrade. The recommended command line parameters are the ```--update all
```

StefanRijnhart commented:
Thanks! I've posted some suggestion and comments, some of which may be subjective so let me know what you think.

jcdrubay commented:
@StefanRijnhart Thanks for your review and suggestions. I have taken them into account.

MiquelRForgeFlow commented on doc/source/migration_details.rst:
```diff
@@ -5,47 +5,62 @@ Check out the code manually and upgrade your database by calling odoo-bin,
 * When installing the openupgradelib make sure you check out the latest version
   from github to get the latest updates and fixes::
 
     pip install git+git://github.com/OCA/openupgradelib.git
```
```suggestion
    pip install git+https://github.com/OCA/openupgradelib.git@master#egg=openupgradelib
```

MiquelRForgeFlow commented on doc/source/migration_details.rst:
````diff
@@ -5,47 +5,62 @@ Check out the code manually and upgrade your database by calling odoo-bin,
+
+Edit the configuration files and command line parameters to point to the
+database you are going to upgrade. The recommended command line parameters are
+the ```--update all --stop-after-init --load=web,openupgrade_framework`` flags.
````
```suggestion
the ```--update all --stop-after-init --load=base,web,openupgrade_framework`` flags.
```

jcdrubay replied:
Is that really necessary?
If it is, then we probably should also update this README as well:

https://github.com/OCA/OpenUpgrade/blob/14.0/openupgrade_framework/README.rst

MiquelRForgeFlow replied:
Doing that here https://github.com/OCA/OpenUpgrade/pull/2629

jcdrubay commented:
Thank you @MiquelRForgeFlow for your review

pedrobaeza commented:
I think this one can be closed as unfolded, isn't it?

jcdrubay commented:
I am not sure what you mean @pedrobaeza  
Also, I think that this PR should be merged even so it only contains minor adjustments.

pedrobaeza commented:
Well, there are conflicts with merged things, so I think there are some kind of overlap. Can you rebase and resolve conflict to see what is remaining?

jcdrubay committed:
Adjust the page migration_details

- With titles to highlight the steps and extra info added within the step
- Add details for installations
- Add some links to module coverage status and development page

jcdrubay commented:
I fixed the merge conflicts and now submitted a new branch with a single commit.

StefanRijnhart commented:
/ocabot merge nobump

pedrobaeza commented:
I think ocabot is not working here.

pedrobaeza commented:
It's working! Good to know.

StefanRijnhart commented:
Maybe it does not work on the older, 'Odoo fork' branches.
